terratest/src/test/pega/pega-tier-config_test.go

```diff
@@ -66,7 +66,7 @@ func VerifyTierConfg(t *testing.T, yamlContent string, options *helm.Options) {
 			pegaConfigMapData := pegaConfigMap.Data
 			compareConfigMapData(t, pegaConfigMapData["prconfig.xml"], "data/expectedInstallDeployPrconfig.xml")
 			compareConfigMapData(t, pegaConfigMapData["context.xml.tmpl"], "data/expectedInstallDeployContext.xml.tmpl")
-			compareConfigMapData(t, pegaConfigMapData["prlog4j2.xml"], "data/expectedInstallDeployPRlog4j2.xml")
+            compareConfigMapData(t, pegaConfigMapData["prlog4j2.xml"], "data/expectedInstallDeployPRlog4j2.xml")
```

Comment:
probably just a mistake, and I don't think it's worth me marking this as "needs work", but looks like this formatting is off